### ISSUE TITLE
model text doesn't show until focus

### ISSUE THREAD

**Issue by alpalmerxactware** [hansottowirtz/simplemde-angular#4]
Just put this into my project. The text in the model doesn't show in the editor until it gains focus.

**Comment by hansottowirtz** [comment]
How do I reproduce this? Haven't experienced it.

**Comment by jkondratowicz** [comment]
I've managed to reproduce this issue, two conditions must be met:
1) content is updated after initialization
2) update happens when the editor is not visible (inside hidden div for example).

Example (I embedded simplemde-angular inline because I couldn't find it on any CDN):

http://jsfiddle.net/jkondratowicz/0p5kyssr/

After 100ms model is updated. After 200ms the parent div is rendered. Editor stays empty until you click on it, then it updates.

**Comment by PabloPG** [comment]
Did you solve the problem?

**Comment by jkondratowicz** [comment]
Nope, we worked around it by not having mde textareas in a collapsed tab. I think this isn't an issue with simplemde-angular, but with one of the underlying libraries.

**Comment by PabloPG** [comment]
I was using "details and summary", I put it to initialize open and then created a 500ms setTimeout to close them. Palliative solution xD
